packages/cli/src/commands/migrate.ts

```diff
@@ -182,13 +183,32 @@ migrateCommand
           title: 'Installing dependencies',
           enabled: (ctx): boolean => !ctx.skip,
           task: async (_ctx, task) => {
+            const packageJson = readJSONSync(resolve(options.basePath, 'package.json'));
             // install custom dependencies
             if (_ctx.userConfig?.hasDependencies) {
               task.title = `Installing custom dependencies`;
               await _ctx.userConfig.install();
             }
+
             // even if typescript is installed, exec this and get the latest patch
             await addDep(['typescript'], true, { cwd: options.basePath });
+
+            // extra dependencies for Ember App/Addon/Engine
+            // TODO: dependes on how much extra stuff we need for a specific framework,
+            // probably need a plugable system for this.
+            if (isEmber(packageJson)) {
```

Comment:
we will need a conditional around which version of Ember and which version of Node. 

Glint only supports:
```
Ember 3.24+
Node 16+
```

Reply:
... and 

`ember-modifier 3.2.7+`

Reply:
Weird to see this as a part of migrate.ts. Isn't better to move this part to a framework-specific package/function (like we did with migration --grapth / migration-graph-ember)?

Move this `['@glint/core', '@glint/template', '@glint/environment-ember-loose']` outside of migrate.ts (params?) and just check `if (additionalDependencies) { ... install them ...}`?

Reply:
@egorio yeah totally agree, that's why in the comment I have the TODO to take it out of the main cli and make it like a plugin-ish structure, still deciding where those should be and I'm open to any suggestions 😄

Reply:
> probably just move it to a new file

Oh, didn't read a comment :) I think it's fine to put in a separate file for now :)